project/src/components/generes-list/genres-list-functions.ts

```diff
@@ -0,0 +1,43 @@
+import { DEFAULT_GENRE } from '../../consts';
+import { Film, FilmCards } from '../../types/film';
+
+export  const createGeneres = (array: FilmCards) => {
```

Comment:
Важное замечание для специалиста со звездочкой.
в функции - createGeneres не нужно массив именно фильмов, а достаточно массив объектов у которых есть единствнное своейство genre
```suggestion
export  const createGeneres = (films: ReadonlyArray<Pick<Film,'genre'>>) => {
```

с помощью такого обозначения типа ты подсказываешь факт "необязательности" других свойств. и что сам массив функция менять (дополнять. опустошать, сортировать) не будет